classes/controller/Controller.php

```diff
@@ -831,4 +832,14 @@ public function getContainer()
     {
         return $this->container;
     }
+
+    /**
+     * Check if multistore feature is enabled.
+     *
+     * @return bool
+     */
+    public function multistoreEnabled(): bool
```

Comment:
Quite often, methods that do return a boolean being with `is` wdyt?

```suggestion
    public function isMultistoreEnabled(): bool
```

Reply:
true 👍 

Anyway, I like the idea of this method, even if it's a one liner, it makes it more readable 👍

Reply:
@matks  @matthieu-rolland 

if(`multistoreEnabled`) vs if(`isMultistoreEnabled`)

Reply:
the second is more readable, no? it's also kind of a standard for boolean methods 

(we might be nitpicking here though)